### PR TITLE
PROD-91: Create 404-page

### PR DESCRIPTION
**Pull request by imysheniuk**
# Reviewers

- [ ] @vikabelous
- [ ] @okohuch
- [x] @Pavliv
- [x] @Retttro

# Jira task
https://ssu-jira.softserveinc.com/browse/PROD-91

# Description
Need to create 404-page according to the site subject

# Checklist:

- [x] Rubocop doesn't raise any warnings and offenses
- [x] All specs pass
- [ ] New features are covered with new specs
- [ ] PR is approved by 2 team mates
- [ ] PR is approved by 2 experts

### CODE REVIEW

File: public/404.html [hunk 1]
```diff
@@ -3,64 +3,16 @@
...
 </head>
 
-<body class="rails-default-error-page">
-  <div class="dialog">
+<body>
+  <div class="error-page">
     <div>
-      <h1>The page you were looking for doesn't exist.</h1>
-      <p>You may have mistyped the address or the page may have moved.</p>
+      <img class="image" src="./prodobro1.png">
```
Comment (Pavliv) [inline public/404.html]: Please add 'alt' attribute to img

File: public/style/404.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  vertical-align: center;
+  border: 3px solid white;
+  margin-top: 10px;
+}
+
+.error-page h1 {
+  font-size: 20vh;
+  font-weight: bold;
+  margin: -6vh 0 0;
+  color:white;
```
Comment (Pavliv) [inline public/style/404.css]: Please add space after ':'. And you can use #fff instead of 'white' like in line 4

File: public/style/404.css [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+.error-page {
+  display: flex;
+  width: 80%;
+  height: 95%;
+  margin: auto;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  vertical-align: center;
+  border: 3px solid white;
```
Comment (Pavliv) [inline public/style/404.css]: Use #fff instead of 'white'

**Review by Pavliv — APPROVED** [review]

**Review by Retttro — APPROVED** [review]

**Comment by viktoriia-bilous** [comment]
Could you add a screenshot of 404 page?

File: public/404.html [hunk 4]
```diff
@@ -3,64 +3,16 @@
...
-<body class="rails-default-error-page">
-  <div class="dialog">
+<body>
+  <div class="error-page">
     <div>
-      <h1>The page you were looking for doesn't exist.</h1>
-      <p>You may have mistyped the address or the page may have moved.</p>
+      <img class="image" src="/prodobro1.png" alt="ProDobro">
+      <h1>404</h1>
+      <p>PAGE IS NOT FOUND</p>
```
Comment (okohuch) [inline public/404.html:14]: I18n?

**Comment by viktoriia-bilous** [comment]
I think you should follow this [approach](https://stackoverflow.com/questions/19103759/rails-4-custom-error-pages-for-404-500-and-where-is-the-default-500-error-mess/25303705#25303705) and create a haml version of 404 page, then you will be able to use I18n

Also, consider adding a button with the link that redirects to the homepage, that's a good UIX practice
Some error pages examples:
https://www.dropbox.com/busindess
https://evernote.com/get-startedd 
https://www.spotify.com/int/why-not-avaidlable/

**Pull request closed without merging**